cmd/kube-proxy/app/BUILD

```diff
@@ -182,21 +182,104 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    srcs = ["server_test.go"],
+    srcs = [
+        "server_test.go",
+    ] + select({
+        "@io_bazel_rules_go//go/platform:android": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:darwin": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:dragonfly": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:freebsd": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:linux": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:nacl": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:netbsd": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:openbsd": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:plan9": [
+            "server_others_test.go",
+        ],
+        "@io_bazel_rules_go//go/platform:solaris": [
+            "server_others_test.go",
+        ],
+        "//conditions:default": [],
+    }),
     embed = [":go_default_library"],
     importpath = "k8s.io/kubernetes/cmd/kube-proxy/app",
     deps = [
         "//pkg/apis/core:go_default_library",
         "//pkg/proxy/apis/kubeproxyconfig:go_default_library",
-        "//pkg/proxy/ipvs:go_default_library",
         "//pkg/util/configz:go_default_library",
-        "//pkg/util/iptables:go_default_library",
         "//pkg/util/pointer:go_default_library",
         "//vendor/github.com/stretchr/testify/assert:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/util/diff:go_default_library",
-        "//vendor/k8s.io/apiserver/pkg/util/feature:go_default_library",
-    ],
+    ] + select({
```

Comment:
Did gazelle generate this?

Reply:
Yes, this is what happens when you have build tags and use hack/update-bazel.sh.